style.css

```diff
@@ -0,0 +1,284 @@
+
+
+h1, h2, h3, h4, h5, h6 {
+    font-family: "Roboto Condensed";
+}
+h1{
+    font-size: 9em;
+    text-align: center;
+    text-transform: uppercase;
+}
+h2{
+    font-size: 5em;
+    text-align: center;
+    text-transform: uppercase;
+    line-height: 4em;
+    color: white;
+    text-shadow: 8px -20px 9px #020819;
+}
+h3{
+    font-size: 4.2em;
+    text-align: center;
+    line-height: 1em;
+}
+h4{
+    font-size: 1.5em;
+    letter-spacing: 0.4px;
+    line-height: 1em;
+    margin-bottom: 15px
+    
+}
+
+h5{
+    font-size: 1.2em;
+    line-height: 1em;
+}
+
+h4{
+    margin: 15px;
+}
+
+.dark-background {
+    background-color: rgb(25, 33, 41);
+    color: #fff;
+}
+.text{
+    font-size: 2em;
+    font-weight: 100;
+    text-align: center;
+    margin: 20px;
+}
+.link-btn{
+    background-color: rgb(67, 163, 230);
+    color: white;
+    font-family: "Roboto Condensed";
+    font-size: 2em;
+    letter-spacing: 0.5px;
+    text-align: center;
+    text-decoration: none;
+    padding: 10px 30px;
+}
+
+.service-box{
+    font-size: 1.7em;
+    text-align: center;
+    border: 2px solid #eee;
+    border-radius: 20px;
+    box-sizing: border-box;
+    box-shadow: 2px 2px 10px 0 #eee;
+    margin: 50px 5% 0 5%;
+    padding: 20px;
+    width: 40%;
+}
+
+
+
+.service-box img{
+    width: 125px;
+}
+
+.container{
+    margin: 0 auto;
+    padding: 75px 0;
+    width: 1200px;
+}
+
+#general-information {
+  display: flex;
+  justify-content: space-evenly;
+  
+}
+
+#general-information .container{
+  display:flex;
+  flex-direction: column;
+  align-items: ceter;
+  width: 370px;
+  
+}
+
+#structure > div{
+  display: flex;
+  justify-content: space-between;
+  flex-direction: row;
+  flex-wrap: wrap;
+  padding: 20 px;
+  margin: 50px 5% 0;
+  
+}
+
+#team > div{
+ diplay: flex;
+ justify-content: space evenly; 
+  flex-direction: row;
+ 
+}
+
+.col-img{
+  width: 30%;
+}
+
+#team > div > article{
+display: flex;
+justify-content: center;
+flex-direction: column;
+align-items: center;
+}
+
+#team .section-text {
+    font-size: 1.9em;
+    text-align: center;
+}
+
+#team .member-name {
+    font-size: 1.5em;
+    font-weight: 700;
+}
+
+#team{
+    padding: 75px0;
+    margin: 0 auto;
+}
+
+#team p{
+    width: 60%;
+    margin: 10px auto;
+}
+
+#team hr{
+    margin: 20px auto 30px;
+    width: 40%;
+}
+
+.schedule-table{
+    margin: 0 auto 40px auto;
+    font-size: 1.5em;
+}
+.schedule-table th, td{
+    border-bottom: solid 1px black;
+    padding: 5px 10px;
+    text-align: center;
+
+}
+
+.slot-form-container{
+  width: 40%;
+  margin: 18px auto 0;
+  font-size: 1.5em;
+  text-align: center; 
+}
+
+.slot-form-container label{
+  line-height: 15px;
+  margin: 5px;
+}
+
+.slot-form-container input, select{
+  height: 20px;
+  outline: none;
+  padding: 3px;
+  width: 100%;
+}
+
+.slot-form-container button{
+  border: 1px solid black;
+  color: black;
+  background-color: white;
+  padding: 9px;
+  margin-top: 15px;
+}
+
+
+.footer{
```

Comment:
You don't need to assign the footer a class, you can also select it by name of the element. The most straightforward selector for me would be:
```
footer > div {
...
}

```